app/Models/Litter/LitterCategory.php

```diff
@@ -43,7 +43,9 @@ public function translate ()
         {
             if ($this->$type)
             {
-                $string .= $this->table . '.' . $type . ' ' . $this->$type . ',';
+                $className = $this->table == 'arts' ? 'art' : $this->table;
+
+                $string .= $className . '.' . $type . ' ' . $this->$type . ',';
```

Comment:
String interpolation would be better here 

```
$string .= "$className.$type.$this->type,";
```

Also, it helps to evaluate expression with ()

```
$x = ($this->table === 'arts') ? 'art' : $this->table;
```

Reply:
This was actually a quick fix more than a proper solution. In the future, we should switch from using the table name to using the snake_cased class name, it's probably better.